python/ray/tests/test_runtime_env_plugin.py

```diff
@@ -52,21 +52,27 @@ def f():
             }
         }).remote()
 
-    output = ray.get(
-        f.options(
-            runtime_env={
-                "plugins": {
-                    MY_PLUGIN_CLASS_PATH: {
-                        "env_value": 42,
-                        "tmp_file": tmp_file_path,
-                        "tmp_content": "hello",
-                        # See https://en.wikipedia.org/wiki/Nice_(Unix)
-                        "prefix_command": "nice -n 19",
+    if os.name != "nt":
+        # Windows does not have a command-line nice
```

Comment:
It would be nice to find a substitute of this that also works on windows (or use a different utility to wrap the worker process in and test that in the test) -- we can do that in a follow up PR.